src/_locales/pt_BR/messages.yml

```diff
@@ -167,27 +173,31 @@ editValueSave:
   message: Salvar
 extDescription:
   description: 'Description for this extension, will be displayed in web store'
-  message: Fornece suporte a scripts de usuários para navegadores.
+  message: Um gerenciador de script de usuário de código aberto que tem suporte à muitos navegadores
 extName:
   description: Name of this extension.
   message: Violentmonkey
 failureReasonBlacklisted:
   description: 'Shown for blacklisted URLs (in the extension icon tooltip, in the popup)'
-  message: ''
+  message: Configurações de Lista negra do Violentmonkey
 failureReasonNoninjectable:
   description: >-
     Shown for URLs that cannot be processed (same places as
     failureBlacklistedUrl)
-  message: ''
+  message: >-
+    Violentmonkey não pode executar scripts de usuários nesta página
+
+(exemplos comuns: interface do navegador, uma extensão, bloqueada via políticas, uma motor de pesquisa
```

Comment:
I guess this line has an incorrect indent too. It has been recognized as a new key since there is a colon in it.

Reply:
Fixed

Reply:
Nothing is changed here, did you commit your changes?

Reply:
Now it's all right? 
You can talk if you need to fix it again.